sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -431,7 +431,7 @@ case class DescribeTableCommand(table: TableIdentifier, isExtended: Boolean, isF
       val schema = DDLUtils.getSchemaFromTableProperties(table)
 
       if (schema.isEmpty) {
-        append(buffer, "# Schema of this table is inferred at runtime", "", "")
+        append(buffer, "# Schema of this table in catalog is corrupted", "", "")
```

Comment:
Do you like the last patch? https://github.com/apache/spark/pull/14148/commits/d92ebcdfd7e525499e0c8b491eeab416ad12ecfd